tests/pytests/addons.py

```diff
@@ -68,6 +68,7 @@ def is_nvidia_gpu_present():
     "simint": psi4.addons("simint"),
     "snsmp2": which_import("snsmp2", return_bool=True),
     "v2rdm_casscf": which_import("v2rdm_casscf", return_bool=True),
+    "qcdb": False,  # capabilities of in-psi and out-of-psi qcdb not aligned
```

Comment:
I don't really understand what this does. Is this just to make sure that `qcdb`-requiring tests should not be run?

Reply:
Seconded.

Reply:
right, this is signaling to pytest that any tests with `using("qcdb")` should plain not run, as I don't want to try to ensure that  qcdb imported from driver and qcdb indep module are exactly the same. In practice, it hits exactly one test, https://github.com/psi4/psi4/blob/master/tests/pytests/test_qcng_dftd3_mp2d.py#L323-L342, so not a big loss.

Reply:
So what is the roadmap on resolving that properly, eventually?

Reply:
I'll probably delete the test. It's a shared test bwn qcng and psi4 that hasn't gotten run much under normal user conditions. Later `using("qcdb")` can be revived when the driver and independent flavors have less overlap, and the shared tests are more extensive.